app/views/step_by_step_pages/schedule.html.erb

```diff
@@ -60,10 +60,28 @@
         ]
       } %>
 
-      <div class="form-group">
-        <%= label_tag "schedule[time]", 'Time' %>
-        <%= text_field_tag "schedule[time]" %>
-      </div>
+      <% time_errors = @step_by_step_page.issues.include?(:schedule_time) ? [
+        {
+          text: @step_by_step_page.issues[:schedule_time],
+        }
+      ] : false %>
+      <%= render "components/autocomplete", {
+        id: "time",
+        name: "schedule[time]",
+        label: {
+          text: t("schedule.edit.time"),
```

Comment:
Where does this text come from?

Reply:
Perhaps this code could be moved to a presenter or a helper class so we can test that we're passing the right things to the component.

Reply:
Good spot, `t("schedule.edit.time"),` shouldn't be here.